File: py/IEC61970/Base/Domain/CostPerVolume.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Fri Dec 15 17:33:10 2023
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Domain.UnitMultiplier import UnitMultiplier
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Domain.UnitSymbol import UnitSymbol
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Domain.Currency import Currency

class CostPerVolume:

    def __init__(self) -> None:
        self.denominator_multiplier: UnitMultiplier = UnitMultiplier()
        self.denominator_unit: UnitSymbol = UnitSymbol.m3  # 'm3'
        self.multiplier: UnitMultiplier = UnitMultiplier.none
        self.unit: Currency = Currency()
        self.value: float = 0.0

File: java/IEC61970/Base/Domain/CostPerVolume.java
package TC57CIM.IEC61970.Base.Domain;


/**
 * Cost per unit volume.
 * @author SELAOST1
 * @version 1.0
 * @created 15-Dec-2023 4:38:26 PM
 */
public class CostPerVolume {

	public static final UnitMultiplier denominatorMultiplier = none;
	public static final UnitSymbol denominatorUnit = m3;
	public UnitMultiplier multiplier;
	public Currency unit;
	public Float value;

	public CostPerVolume(){

	}

	public void finalize() throws Throwable {

	}

}